mailpoet/tests/performance/config.js

```diff
@@ -9,5 +9,9 @@ export const adminUsername = 'admin';
 export const adminPassword = 'password';
```

Comment:
I was thinking maybe we could rename this file to `config.sample.js`, just like what we do for `.env` file, and ask the devs to copy it to `.config.js`, especially since we'll need to update and add the correct password, this will make sure we're not committing the password to the repo by mistake.

Reply:
Hi @samnajian , thanks for reviewing. I understand the point, but the password should always be `password` (maybe in the future if we will have some external site with some password). In one of the next PRs I will add data to a local site so we can test against local always.